atlas_admin/helpers.py

```diff
@@ -280,22 +280,33 @@ def userInstanceLookup(instanceList):
     return uniqueUserNameList
 
 
-def summaryStatistics():
-    # TODO drupal_system_status
-    q = get_internal('statistics')
+def getAllResults(atlasType, **findThisElement):
```

Comment:
There are some other places where this function can be used

Reply:
Yes, BUT ... this ticket is 'Add Theme List to Instance Summary Page.' I'd rather make another ticket for refactoring all the code.